pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Topic.java

```diff
@@ -144,13 +144,17 @@ default void setEntryTimestamp(long entryTimestamp) {
 
     void removeProducer(Producer producer);
 
+    /**
+     * return if transaction disabled when call checkIfTransactionBufferRecoverCompletely
+     */
+    CompletableFuture<Void> TRANSACTION_DISABLED = CompletableFuture.completedFuture(null);
```

Comment:
I feel that this variable definition in class `Topic` is not reasonable, and the naming is not easy to understand

Reply:
thanks for review, i agree with you, i will move the defination to `TransactionBuffer`